content/applications/productivity/calendar.rst

```diff
@@ -9,3 +10,208 @@ Calendar
 
    calendar/outlook
    calendar/google
+
+Odoo Calendar is a scheduling app that allows users to integrate a company’s business flow into a
+single management platform. By integrating with the other apps in Odoo’s ecosystem, Calendar allows
+users to schedule meetings, organize meetings, schedule events, plan employee appraisals,
+coordinate projects, and more – all from the same platform.
+
+In the **Calendar** app, users have an overview of their current meetings. The selected view option
+will appear as a :guilabel:`Day`, :guilabel:`Week`, :guilabel:`Month`, or :guilabel:`Year` button.
+Under the view options, users can also enable or disable :guilabel:`Show weekends`.
+
+.. image:: calendar/calendar-overview.png
+   :align: center
+   :alt: Overview of Calendar app.
+
+.. tip::
+   Depending on the selected view option, users can use the arrow buttons to switch between days,
+   weeks, etc., then switch back to the current day with the :guilabel:`Today` button.
+
+Sync third-party calendars
+--------------------------
+
+Users can sync Odoo with existing :doc:`Outlook <calendar/outlook>` and/or :doc:`Google
+<calendar/google>` calendars by heading to :menuselection:`Calendar --> Configuration --> Settings`.
+Events created in synced calendars will automatically appear across the integrated platforms.
+
+Create Calendar activities from chatter
+---------------------------------------
+
+Instantly create new meetings anywhere in Odoo through an individual record’s chatter, like
+in a CRM opportunity card or Project Task.
+
+From chatter, click on the :guilabel:`Activities` button. In the new :guilabel:`Schedule Acitivity`
+window, select :guilabel:`Activity Type`, which will populate a set of smart buttons depending on
+the activity. Activities that would involve other attendees' schedules like :guilabel:`Meeting` or
+:guilabel:`Call for Demo` will link to **Calendar** app.
+
+.. image:: calendar/calendar-chatter-activity-type-call-for-demo.png
+   :align: center
+   :alt: Schedule an Activity window on CRM app: Schedule a call for demo.
+
+Plan an event
+-------------
+
+To put an event on the calendar, click into the target date. Under **New Event** in the new window,
+add the event title.
+
+.. image:: calendar/calendar-schedule-event.png
+   :align: center
+   :alt: Schedule an event window on Calendar app.
+
+The target date auto-populates in the :guilabel:`Start` section. This can be changed by clicking
+into the date section and selecting a date from the calendar. For multi-day events, select the end
+date on the right calendar, then click the :guilabel:`Apply` button.
+
+Additional event information can also be configured:
```

Comment:
I think you can delete this line, the explanations of other ways to configure events speak for themselves! :cowboy_hat_face: